packages/strapi-design-system/src/DatePicker/DatePicker.js

```diff
@@ -68,49 +96,69 @@ export const DatePicker = ({ initialDate, selectedDate, onChange, label, selecte
         >
           {visible && (
             <FocusTrap onEscape={() => setVisible(false)}>
-              <RawTable
-                colCount={7}
-                rowCount={weeks.length + 1}
-                initialCol={activeCol}
-                initialRow={activeRow}
-                role="grid"
-              >
-                <RawThead>
-                  <RawTr>
-                    <DatePickerTh>{sun}</DatePickerTh>
-                    <DatePickerTh>{mon}</DatePickerTh>
-                    <DatePickerTh>{tue}</DatePickerTh>
-                    <DatePickerTh>{wed}</DatePickerTh>
-                    <DatePickerTh>{thu}</DatePickerTh>
-                    <DatePickerTh>{fri}</DatePickerTh>
-                    <DatePickerTh>{sat}</DatePickerTh>
-                  </RawTr>
-                </RawThead>
-                <RawTbody>
-                  {weeks.map((week, index) => (
-                    <RawTr key={`week-${index}`}>
-                      {week.map(({ date, outsideMonth, isSelected }) => {
-                        return (
-                          <DatePickerTd
-                            key={`${date.getFullYear()}-${date.getMonth() + 1}-${date.getDate()}`}
-                            outsideMonth={outsideMonth}
-                            onSelectDay={() => handleSelectDay(date)}
-                            isSelected={isSelected}
-                          >
-                            <span aria-hidden={true}>{date.getDate()}</span>
-                            <VisuallyHidden>
-                              <span>{langFormatter.format(date)}</span>
-                            </VisuallyHidden>
-                          </DatePickerTd>
-                        );
-                      })}
+              <Box padding={4}>
+                <Box paddingBottom={4}>
+                  <Row>
+                    <SimpleMenu id="year" label={months[date.getMonth()]}>
+                      {months.map((month) => (
+                        <MenuItem key={month} onClick={() => handleMonthChange(month)}>
+                          {month}
+                        </MenuItem>
+                      ))}
+                    </SimpleMenu>
+                    <SimpleMenu id="months" label={date.getFullYear()}>
```

Comment:
There's a proptypes error on this I think (number and string)

Reply:
Nice catch ! thanks